types/webextension-polyfill/.eslintrc.json

```diff
@@ -1,5 +1,8 @@
 {
     "rules": {
+        "@definitelytyped/export-just-namespace": "off",
+        "@typescript-eslint/no-invalid-void-type": "off",
+        "@definitelytyped/no-unnecessary-generics": "off",
```

Comment:
Fix the errors or disable it on the line instead of disabling for the whole package

Reply:
That is a bit hard to implement for code that is generated from a JSON definition. Is that really necessary?

Reply:
Hi @sheetalkamat, I've changed the generator to remove some of the global linting rules, but making it add a per-line disable for "@definitelytyped/no-single-element-tuple-type" would be very hard to implement, as the function getType, which returns it (a single string), has a huge call hierarchy.

I can only assure you, that the generator will never produce these tuple-type without reason. The point of these manual disables is that we have to think about them, but if the generator produces them, then there's really not much point to them.

I hope it's okay like this.